components/header.tsx

```diff
@@ -1,28 +1,69 @@
-import React from "react";
-import styles from "../styles/Header.module.css";
-import Link from "next/link";
+import React from 'react';
+import styles from '../styles/Header.module.css';
+import Link from 'next/link';
+import { Button } from '@mui/material';
+import KeyboardBackspaceIcon from '@mui/icons-material/KeyboardBackspace';
 
 interface Headerprops {
   data: any;
   taskNumber?: number;
   description?: string;
+  page?: string;
```

Comment:
Why is `page` optional? It seems like it will be used in all cases:)

Reply:
I see that it is renamed in #118 so it is easier to potentially change there anyway